### ISSUE TITLE
WDIO tests are failing for Terra-Clinical-Data-Grid

### ISSUE THREAD

**Issue by supreethmr** [cerner/terra-clinical#524]
# Issue Description
WDIO tests are failing for Terra-Clinical-Data-Grid due to below accessibility violations 
```
[medium] DataGrid with pinned columns and overflow columns with initial rendering [#pinned-and-overflow] is accessible and is within the mismatch tolerance:
expected no accessibility violations but got:
	{
  "description": "Ensures aria-hidden elements do not contain focusable elements",
  "help": "ARIA hidden element must not contain focusable elements",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=axeAPI",
  "id": "aria-hidden-focus",
  "impact": "serious",
  "nodes": [
    {
      "all": [
        {
          "data": null,
          "id": "focusable-not-tabbable",
          "impact": "serious",
          "message": "Focusable content should have tabindex='-1' or be removed from the DOM",
          "relatedNodes": [
            {
              "html": "<div role=\"button\" aria-hidden=\"true\" class=\"DataGrid-module__leading-focus-anchor___2JPRS\" tabindex=\"0\"></div>",
              "target": [
                ".DataGrid-module__leading-focus-anchor___2JPRS"
              ]
            }
          ]
        }
      ],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Focusable content should have tabindex='-1' or be removed from the DOM",
      "html": "<div role=\"button\" aria-hidden=\"true\" class=\"DataGrid-module__leading-focus-anchor___2JPRS\" tabindex=\"0\"></div>",
      "impact": "serious",
      "none": [],
      "target": [
        ".DataGrid-module__leading-focus-anchor___2JPRS"
      ]
    }, 
```

Recently New axe rules : [wcag2a', 'wcag21aa'](https://github.com/cerner/terra-toolkit/pull/303/files) were added to toolkit and relased with toolkit version 5.1.0 . 

New rules are catching this accessibility violations of clinical-data-grid, which were not caught before with previous axe rules : ['wcag2aa', 'section508'] .

## Additional Context / Screenshots
Travis Build job of Terra-clinical(master) :  https://travis-ci.com/cerner/terra-clinical/jobs/210876148

**Comment by bjankord** [comment]
The following PR resolve the issue: https://github.com/cerner/terra-clinical/pull/522
We are waiting for translations to before we merge the PR.
We should have translations back this Friday.

**Comment by nramamurth** [comment]
I am closing this issue since #522 resolved it.
